### PR TITLE
Added skip xml environment value documentation [9409]

### PR DESCRIPTION
**Pull request by imontesino**
Adds documentation for added feature in eprosima/Fast-DDS#1350.
Signed-off-by: imontesino <monte.igna@gmail.com>

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/617/)

File: docs/fastdds/env_vars/env_vars.rst [hunk 1]
```diff
@@ -67,4 +67,22 @@ This is the list of environment variables that affect the behavior of *Fast DDS*
...
+    | .. code-block:: bash                                             |
+    |                                                                  |
+    |    export SKIP_DEFAULT_XML=1                                     |
+    +------------------------------------------------------------------+
+    | **Windows**                                                      |
+    +------------------------------------------------------------------+
+    | .. code-block:: bash                                             |
+    |                                                                  |
+    |    set SKIP_DEFAULT_XML=1                                        |
+    +------------------------------------------------------------------+
```
Comment (MiguelCompany) [inline docs/fastdds/env_vars/env_vars.rst:88]:
```suggestion
    +------------------------------------------------------------------+

```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Just to keep linters happy

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/618/)

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**